.github/workflows/node.js.yml

```
@@ -0,0 +1,30 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [9.x, 14.x, 16.x, 18.x]
```

```suggestion
        node-version: [9.x, 14.x, 16.x, 18.x]
        fail-fast: false
```

By default, if one test fails, GitHub Actions aborts all other tests in the matrix, so `fail-fast: true`.

I think it's better to complete all tests. Then you can see if the problem is with a specific version of Node.js.

If you apply this suggestion then GitHub Actions runs all tests to completion.